pkg/cmd/cli/cmd/set/env.go

```diff
@@ -48,6 +48,12 @@ syntax.`
   # Update all containers in all replication controllers in the project to have ENV=prod
   %[1]s env rc --all ENV=prod
 
+  # Import environment from a secret
+  %[1]s env --from-secret=mysecret dc/myapp
```

Comment:
So how would the difference between secrets injected as base64 or
regular string be handled?